bin/update-versions

```diff
@@ -57,6 +57,10 @@ function readDistroPackageVersion($url, $distro)
       $xpath = '//*[@id="content"]/h1';
       $regex = '/\(([^\)]+)/';
     break;
+    case 'centos':
+      $xpath = '//a[matches(@href, "^php-\d+")]';
```

Comment:
Sorry, this expression breaks XPath parser.

Maybe regexp unsupported by parser version, looking for write a fix.